### ISSUE TITLE
How much time did it take?

### ISSUE THREAD

**Issue by DenisSergeevitch** [janivanecky/Artistic-Style#3]
How much time did it take to generate one image? Is it faster than https://github.com/OlavHN/fast-neural-style ?

**Comment by janivanecky** [comment]
No, I'm using original algorithm proposed by [Gatys _et al_](http://arxiv.org/abs/1508.06576) which means image is generated in multiple iterations. In case of my implementation I used 500 iterations which takes on Tesla K20m ~5 minutes.

**Comment by ycjing** [comment]
@DenisSergeevitch Hi, did you run the code in  https://github.com/OlavHN/fast-neural-style ? How much time did it take for one stylized image? 
(Is it approximately the time described in the paper, which is 0.015s for 256 \* 256, 0.05s for 512 \* 512 and 0.21s for 1024 \* 1024 on GTX Titan X?)
